.github/workflows/build.yml

```diff
@@ -0,0 +1,75 @@
+name: Test and Build Workflow
+# This workflow is triggered on pull requests to main branch
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    # Job name
+    name: Build Replication plugin
+    runs-on: ubuntu-latest
+    steps:
+      # This step uses the setup-java Github action: https://github.com/actions/setup-java
+      - name: Set Up JDK 14
+        uses: actions/setup-java@v1
+        with:
+          java-version: 14
```

Comment:
This is standard across other opensearch repos as well right?

Reply:
Yes this is the same across other plugins as well.

Reply:
Ack. Will push this changes and try to build the plugin with jdk 11 in a separate PR.